tests/Unit/MartinGeorgiev/Utils/PostgresArrayToPHPArrayTransformerTest.php

```diff
@@ -144,7 +144,7 @@ public static function provideValidTransformations(): array
             ],
             'github #351 regression: string with special characters and backslash' => [
                 'phpValue' => ["!@#\\$%^&*()_+=-}{[]|\":;'\\?><,./"],
-                'postgresValue' => '{"!@#\$%^&*()_+=-}{[]|\":;\'\?><,./"}',
+                'postgresValue' => '{"!@#\\\\$%^&*()_+=-}{[]|\":;\'\\\\?><,./"}',
```

Comment:
This is the real value of PostgreSQL, as demonstrated by simulated integration tests.